apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.CommonTesting/SpannerTestDatabase.cs

```diff
@@ -79,6 +83,9 @@ private SpannerTestDatabase(string projectId)
             TestLogger.Install();
 
             ProjectId = projectId;
+
+            MaybeInitializeInstance();
```

Comment:
We currently rely on there being a `spannerintegration` instance available in the project. But if we're using the emulator, there are no default instances available and we need to create one.

Reply:
Now using gcloud in the script so I was able to create an instance on the command line. Reverted these commits.